### ISSUE TITLE
Playlist().length Value Error 

### ISSUE THREAD

**Issue by mastermach50** [pytube/pytube#1008]
Hello There,
I am unable to print the number of videos in a playlist using Playlist().length.
This is my code.
```python
from pytube import Playlist

yt = Playlist("https://www.youtube.com/playlist?list=UUeZje_7vr6CPK9vPQDfV3WA")

print(yt.length)
```
## Info
This is the error : 
`ValueError: invalid literal for int() with base 10: '1,354'`
possibly due to the comma.

The total size of the playlist is 1354 videos and I am guessing that this value was returned from youtube as a string which had a comma.

Tried the code with again by downloading the master branch and importing it, did not work.

Python version - 3.8.8

**Comment by tfdahlin** [comment]
This definitely sounds like an error with how the string gets parsed. I'm surprised this didn't get caught sooner, I'll fix it

**Comment by tfdahlin** [comment]
Fixed in v10.8.4, you should be able to update with `python -m pip install --upgrade pytube`
